scripts/feature_tests/upgrade/upgrade_common.sh

```diff
@@ -144,17 +148,17 @@ EOF
 }
 
 function manage_node_taints() {
-    kubectl get secrets "${CLUSTER_NAME}"-kubeconfig -n "${NAMESPACE}" -o json | \
-      jq -r '.data.value'| base64 -d > /tmp/kubeconfig-"${CLUSTER_NAME}".yaml
+    set_kubeconfig_towards_target_cluster
```

Comment:
Calling this function multiple times, specially after pivoting has an undesired result.
Eg. after pivoting the secret does not exist on the source cluster which results in an empty kubeconfig file

Reply:
fixed